marimba-web/Models/Term.cs

```diff
@@ -9,10 +9,39 @@ public class Term
     {
         public string name { get; set; } // i.e. W2020
         public IList<Member> members { get; set; }
-        public IList<Member> limboMembers { get; set; }
+        public IList<Member> limboMembers { get; set; } // Subset of members
```

Comment:
Do you think it'd be better to have this list be a list/set of GUIDs, instead of another copy of the member?  We can add a GetMemberFromGUID() method or something to return the member in the Term or Club classes (or both).

Reply:
If we do this then we should also have a guid-member dictionary for more efficient lookup by ID. It would make things like constructing elector lists easier but idk how this change would impact other scenarios.

Reply:
Good point.  A lot of things will just iterate through it, and we'll need to implement a search feature by name, email, student #, etc., too.  I do see some places in the current marimba that get the member index in the term, though.  What do you think about having a private dictionary field that's only accessed with a getMemberFromGuid() method or similar?

I was thinking that the limbo structure would be a subset of the member structure (like how it currently is).

Reply:
Sure, what are your thoughts on the updated changes?

Reply:
I think it looks good.  Thanks for the work!